src/askscript/__tests__/09-operators/operators-10-mod-oper.test.ts

```diff
@@ -0,0 +1,11 @@
+import { resources } from '../../../askvm';
+import { e2e } from '../../../utils/tools';
+
+test('mod operator', async () => {
+  const environment = { resources };
+  await expect(e2e('1998%9', environment)).resolves.toBe(0);
```

Comment:
Your tests are good, could you just add spaces around `%` signs to improve readability?

Reply:
> package.json and playgroundBuild.ts shouldn't be changed in this PR as these are changes unrelated to modulo
The issue was that my forked repo was not sync with xFAANG/askql. 
> Your tests are good, could you just add spaces around % signs to improve readability?
done that.